x-pack/legacy/plugins/siem/index.ts

```diff
@@ -155,6 +156,9 @@ export const siem = (kibana: any) => {
       const initializerContext = { ...coreContext, env } as PluginInitializerContext;
       const serverFacade = {
         config,
+        encryptionKeyRandomlyGenerated:
+          get('encryptionKeyRandomlyGenerated', newPlatform.setup.plugins.encryptedSavedObjects) ??
+          false,
```

Comment:
I know exactly why you have to do this and later we really need to see when they can change NP to allow for correct types rather than `object`. I have the same issues with alertClient and actionsClient.

No changes requested, just an observation.